fc/language/expressions/array.py

```diff
@@ -71,20 +40,20 @@ def Interpret(self, env):
     def DevelopResultWithCompile(self, range_name, ranges, compiled_gen_expr, env):
```

Comment:
Are we sticking with CamelCase for this one? (Actually it's PascalCase isn't it?)

Reply:
I didn't think it was worth going through and changing everything away from the Chaste convention!